src/services/messagesService.js

```diff
@@ -72,15 +87,15 @@ const cancelableLookForNewMessages = function() {
 		} catch (exception) {
 			if (axios.isCancel(exception)) {
 				console.debug(exception.message)
-				return exception.message
+				throw exception
```

Comment:
You should not catch at all.
No try...catch here

Reply:
https://github.com/nextcloud/gallery/blob/refactor/gallery-19/src/services/FolderInfo.js